google-cloud-storage/src/main/java/com/google/cloud/storage/Channel.java

```diff
@@ -0,0 +1,294 @@
+  }
+
+  /* Returns an arbitrary string delivered to the target address with each notification delivered over the channel.*/
+  public String getToken() {
+    return token;
+  }
+
+  /* Returns the notification channel expiration date and time.*/
+  public Long getExpiration() {
+    return expiration;
+  }
+
+  /* Returns the type of delivery mechanism used for this channel.*/
+  public String getType() {
+    return type;
+  }
+
+  /* Returns the address where notifications are delivered.*/
+  public String getAddress() {
+    return address;
+  }
+
+  /* Returns the newly declared parameter by name.*/
+  public Map<String, String> getParams() {
+    return params;
+  }
+
+  /* Returns a Boolean value to indicate whether payload is wanted.*/
+  public Boolean getPayload() {
+    return payload;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    Channel channel = (Channel) o;
+    return Objects.equals(kind, channel.kind)
+        && Objects.equals(id, channel.id)
+        && Objects.equals(resourceId, channel.resourceId)
+        && Objects.equals(resourceUri, channel.resourceUri)
+        && Objects.equals(token, channel.token)
+        && Objects.equals(expiration, channel.expiration)
+        && Objects.equals(type, channel.type)
+        && Objects.equals(address, channel.address)
+        && Objects.equals(params, channel.params)
+        && Objects.equals(payload, channel.payload);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(
+        kind, id, resourceId, resourceUri, token, expiration, type, address, params, payload);
+  }
+
+  /** Returns a builder for the {@link Channel} object. */
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  /** Returns a builder for the {@link Channel} object. */
+  public Builder toBuilder() {
+    return new Builder(this);
+  }
+
+  com.google.api.services.storage.model.Channel toProtobuf() {
+    com.google.api.services.storage.model.Channel channel =
+        new com.google.api.services.storage.model.Channel();
+    if (kind != null) {
+      channel.setKind(kind);
+    }
+    if (id != null) {
+      channel.setId(id);
+    }
+    if (resourceId != null) {
+      channel.setResourceId(resourceId);
+    }
+    if (resourceUri != null) {
+      channel.setResourceUri(resourceUri);
+    }
+    if (token != null) {
+      channel.setToken(token);
+    }
+    if (expiration != null) {
+      channel.setExpiration(expiration);
+    }
+    if (type != null) {
+      channel.setType(type);
+    }
+    if (address != null) {
+      channel.setAddress(address);
+    }
+    if (params != null) {
+      channel.setParams(params);
+    }
+    if (payload != null) {
+      channel.setPayload(payload);
+    }
+    return channel;
+  }
+
+  static Channel fromProtobuf(com.google.api.services.storage.model.Channel channel) {
```

Comment:
same here

Reply:
same here `fromProtobuf()` instead of `fromPb()`

Reply:
I agree, to/fromProtobuf() is better than to/fromPb(). Google style doesn't allow acronyms.